modules/ROOT/pages/_partials/commons/common-vector-search.adoc

```diff
@@ -0,0 +1,217 @@
+
+See here for examples of how to xref:{param-module}:working-with-vector-search.adoc#create-a-lazy-vector-index[use lazy index in your applications].
+
+Lazy indexing is an alternate approach to using the standard predictive model with regular vector indexes which handle the indexing process automatically. 
+The table below compares the two processes.
+
+.Regular Vector Indexes vs Lazy Vector Indexes
+[cols ="3*"]
+|===
+|Feature |Regular Index |Lazy Index
+
+|Update when documents are changed
+|image:ROOT:yes.png[]
+|image:ROOT:no.png[]
+
+|Update when documents are deleted or purged
+|image:ROOT:yes.png[]
+|image:ROOT:yes.png[]
+
+|The application has control when to update the index
+|image:ROOT:no.png[]
+|image:ROOT:yes.png[]
+
+|The application can skip updating the index
+|image:ROOT:no.png[]
+|image:ROOT:yes.png[]
+
+|Is an asynchronous process
+|image:ROOT:no.png[]
+|image:ROOT:yes.png[]
+
+|===
+
+=== About Vector Encoding
+
+Vector encoding reduces the size of Vectors index by algorithmic compression.
+You can configure the Vector Encoding in Couchbase Lite depending on your application's needs.
+
+This vector encoding compression reduces disk space required and I/O time during indexing and queries, but greater compression can result in inaccurate results in distance calculations.
+
+Vector Search for Couchbase Lite supports the following encoding algorithms:
+
+* None - This will return the highest quality results but at high performance and disk space costs
+
+* Scalar Quantizer - This reduces the number of bits used for each number in a vector. 
+The number of bits per component can be set to 4, 6, or 8 bits.
+The default setting in Couchbase Lite is 8 bits Scalar Quantizer or SQ-8.
+
+* Product Quantizer - This reduces the number of dimensions and bits per dimension. 
+It splits the vectors into multiple subspaces and performing scalar quantization on each space independently before compression.
+This can produce higher quality results than Scalar Quantization at the cost of greater complexity.
+
+NOTE: Quantizers are algorithmic processes that map input values from a larger set to output values in a smaller set, common quantization processes can include operations such as rounding and truncation.
+
+=== About Centroids
+
+Centroids are vectors that function as the center point of a vector cluster within the data set.
+Each vector is then associated to the vector it is closest to by https://en.wikipedia.org/wiki/K-means_clustering[k-means clustering.]
+Each Centroid is contained within a bucket along with its associated vectors, the greater amount of Centroids, the greater the potential accuracy of the model.
+This will also incur a longer indexing time.
+
+Choosing Centroids in vector search involves trade-offs that can impact clustering effectiveness and search efficiency. The initial selection of Centroids, the number chosen, and their sensitivity to high dimensionality and outliers affect the quality of vector clustering.
```

Comment:
i-cap Vector Search when referring to the feature.

Reply:
Good spot! Corrected throughout